src/EditorFeatures/CSharpTest/ConvertToRecord/ConvertToRecordTests.cs

```diff
@@ -0,0 +1,864 @@
+    {
+    }
+}
+";
+            await TestRefactoringAsync(initialMarkup, changedMarkup).ConfigureAwait(false);
+        }
+
+        [Fact]
+        public async Task TestMovePropertiesAndDeleteSimpleEqualOperators1()
+        {
+            var initialMarkup = @"
+namespace N
+{
+    public class [|C|]
+    {
+        public int P { get; init; }
+        public bool B { get; init; }
+
+        public static bool operator ==(C c1, object? c2) {
+            return c1.Equals(c2);
+        }
+
+        public static bool operator !=(C c1, object? c2) {
+            return !(c1 == c2);
+        }
+    }
+}
+";
+            var changedMarkup = @"
+namespace N
+{
+    public record C(int P, bool B)
+    {
+    }
+}
+";
+            await TestRefactoringAsync(initialMarkup, changedMarkup).ConfigureAwait(false);
+        }
+
+        [Fact]
+        public async Task TestMovePropertiesAndKeepSideEffectOperator()
+        {
+            var initialMarkup = @"
+using System;
+
+namespace N
+{
+    public class [|C|]
+    {
+        public int P { get; init; }
+        public bool B { get; init; }
+
+        public static bool operator ==(C c1, object? c2)
+        {
+            Console.WriteLine(""checking equality"");
+            return c1.Equals(c2);
+        }
+
+        public static bool operator !=(C c1, object? c2)
+        {
+            return !(c1 == c2);
+        }
+    }
+}
+";
+            var changedMarkup = @"
+using System;
+
+namespace N
+{
+    public record C(int P, bool B)
+    {
+        public static bool operator ==(C c1, object? c2)
+        {
+            Console.WriteLine(""checking equality"");
+            return c1.Equals(c2);
+        }
+
+        public static bool operator !=(C c1, object? c2)
+        {
+            return !(c1 == c2);
+        }
+    }
+}
+";
+            await TestRefactoringAsync(initialMarkup, changedMarkup).ConfigureAwait(false);
+        }
+
+        [Fact]
+        public async Task TestMovePropertiesWithSimpleDocComments()
+        {
+            var initialMarkup = @"
+namespace N
+{
+
+    /// <summary>
+    /// some summary
+    /// </summary>
+    public class [|C|]
+    {
+
+        /// <summary>
+        /// P is an int
+        /// </summary>
+        public int P { get; init; }
+
+        /// <summary>
+        /// B is a bool
+        /// </summary>
+        public bool B { get; init; }
+    }
+}
+";
+            var changedMarkup = @"
+namespace N
+{
+    /// <summary>
+    /// some summary
+    /// </summary>
+    /// <param name=""P""> P is an int </param>
+    /// <param name=""B""> B is a bool </param>
+    public record C(int P, bool B)
+    {
+    }
+}
+";
+            await TestRefactoringAsync(initialMarkup, changedMarkup).ConfigureAwait(false);
+        }
+
+        [ConditionalFact(AlwaysSkip = "Multiline doc comment formatting isn't working")]
```

Comment:
i would just have the test show what our behavior is.  we can then have a tracking bug if we want to improve something here.